pages/dkp/konvoy/2.2/release-notes/index.md

```diff
@@ -115,6 +115,29 @@ When upgrading to this release, the following services and service components ar
 
 ## Known issues
 
+### Work around to upgrade from 1.8.x > 2.1.x > 2.2.0 
```

Comment:
This makes it seem like it's a workaround for a known issue instead of a pre-planned step that's required and documented as part of this major-version upgrade. Which story do we want to tell?

Reply:
Apologies... Doc was told by Carter it was in "issue" getting fixed in 2.3.  If this is just how it works from here on out, we will certainly put that in the correct area of documentation regarding Upgrades.

Reply:
Removed from RN